desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -303,6 +303,7 @@ def cancel(self, notebook, snippet):
           pass
 
         connection['connection'].close()
```

Comment:
FAR: code duplication in close_statement for close, dispose, del. For future reviews.